docker/ops.go

```diff
@@ -186,39 +126,19 @@ func ProcessMessage(message *babashka.Message) (any, error) {
 				return nil, err
 			}
 			return shellquote.Split(args[0])
-		case "docker.tools/generate-sbom":
```

Comment:
Should be removing the old generate-sbom? This could be useful IMHO.

Reply:
We either need to remove it or do a load of extra work. `generate-sbom` is using the old `index-cli-plugin` which would need updating in order to be able to import alongside `scout-cli-plugin.

Reply:
I've been using generate-sbom to transact datoms against a local store but I understand if this is no longer compatible or supported.